Junction/CustomOption.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  CustomOption.swift
+//  Junction
+//
+//  Created by Jimmy McDermott on 8/10/16.
+//  Copyright © 2016 BlueLine Labs. All rights reserved.
+//
+
+import Foundation
+
+public enum CustomOption {
+    case None
+    case Custom(String)
```

Comment:
It's not really clear how this string will be used. Maybe do a typealias with something like TextFieldPlaceholder?